project_generation/helpers.go

```diff
@@ -408,15 +408,47 @@ func runGoModTidy(ctx context.Context, pathToRepo string) {
 	}
 }
 
+type AppOptions struct {
+	Type ProjectType
+}
+
 // FinaliseModules will run go build ./... to generate go modules dependency management files
-func FinaliseModules(ctx context.Context, pathToRepo string) {
+func FinaliseModules(ctx context.Context, pathToRepo string, opts ...AppOptions) {
 	runGoModTidy(ctx, pathToRepo)
-	cmd := exec.Command("go", "build", "./...")
+
+	if len(opts) > 0 && opts[0].Type == Controller {
+		installGoBinData(ctx, pathToRepo)
+		return
+	}
```

Comment:
Urgh! The real answer is to remove use of bindata, but that's too big for this bit of work. It's quite possible, but the way the frontend controller code uses bindata concepts is a little coupled and messy so needs a little work before we can remove this messy workaround.

Reply:
Yeah I don't plan on doing that piece of work here - never my intention to get this deep into the generation tool; can be another maintenance ticket in the red backlog board